docs/source/quickstart.md

```diff
@@ -20,13 +20,13 @@ $ livepeer -network rinkeby -broadcaster
 
 *Note that if you are already running an orchestrator node on the same machine, you will also have to pass additional flags into this command to specify unique ports so as not to conflict with your orchestrator node. See the below section on testing your transcoding setup for more detail.*
 
-### Getting test ETH
+## Getting test ETH
 
-If you are connecting to the Rinkeby public testnet you can get test ETH from the [Rinkeby faucet](https://faucet.rinkeby.io/).
```

Comment:
I'd keep the mention of the Rinkeby public testnet to make it clear that the "Rinkeby faucet" is only applicable if you are connecting to the testnet.

Reply:
Didn't mean to remove this, didn't update my master at first so this probably got removed during rebasing on top of updated master.